src/Domain/Entities/Products/Product.cs

```diff
@@ -5,9 +5,9 @@ namespace CleanCompanyName.DDDMicroservice.Domain.Entities.Products;
 
 public class Product : AuditableEntity
 {
-    public Guid Id { get; init; }
+    public ProductId Id { get; init; }
     public string Sku { get; init; } = default!;
```

Comment:
Could be but as the template just needs to give some examples maybe is better not to overuse the value object.
But maybe you're right and it is better to act as if the template was going to solve some real use case. 
I don't know...
The SKU one could be very similar to the title, but maybe it could be worth it as an example to do the value object for the price as this can be money and the money has very special properties as the currency and an amount that cannot be negative (normally).

Reply:
Agree on all said ;) Let's keep it as it is :)